src/Apps/CollectorProfile/collectorProfileRoutes.tsx

```diff
@@ -292,4 +390,147 @@ export const collectorProfileRoutes: AppRouteConfig[] = [
       PriceEstimateConfirmation.preload()
     },
   },
+  {
+    path: "/collector-profile/my-collection/submission",
```

Comment:
I am not a big fan of having a new submission route here - I understand we did that in the past but I think this has led to issues with data since they will have to take into account the new route and if we deep link into this route, it will break in app. Would it be possible to explore ways where we only have one sell route regardless from where you start your submission?

I understand this might not be a direct thing to pull here but I would suggest spiking on this given that we are going to be touching this surface in a few weeks time when we port the new submission flow to web. You can potentially ticket this and bring it to be discussed with the team over standup or refinement

Reply:
You can as well do it in this PR since I see you already removed the 2 out the 4 routes that get you to consign. Only 1 left to remove

Reply:
That's a good point. We can definitely use /sell instead of /submission here, we just need to talk to the data team first. But I'm not sure if removing /collector-profile/my-collection/ is a good idea if this is what you mean. Having /collector-profile/my-collection/ maintains a cohesive experience for my collection (compare to add/edit artwork) and back button behaviour. 
Please, let me know if I misunderstood you

Reply:
Actually having only /sell/submission/SUBMISSION-STEP/{artworkId} would make total sense.Is that what you mean?